src/network/Proxy.ts

```diff
@@ -405,18 +496,18 @@ class ForwardProxy {
       this.logger.info(`Handled CONNECT to ${ingressAddress}`);
     } finally {
       release();
-      this.connectionLocks.delete(ingressAddress as Address);
+      this.forwardConnectionLocks.delete(ingressAddress as Address);
```

Comment:
Instead of prepending `forward`, to match the name of the classes `ConnectionForward`, use `this.connectionsForward`.

This goes for all the other methods and variable names as well.

Reply:
Same with `reverse`.

Reply:
The reason for this naming order when `Y` extends `X`, I call them:

```
class X
class XY
```

But if instead I have a class that builds on tops/encapsulates similar behaviour I'll use prepending naming:

```
class AX
class BX
```

Case in point:

* `ConnectionForward` extends `Connection`
* `ConnectionReverse` extends `Connection`

While:

* `ForwardProxy`
* `ReverseProxy`

Are variants of a "proxy" behaviour.

Reply:
@emmacasolin good to know. I'll move this into the wiki.